vizro-core/src/vizro/static/css/layout.css

```diff
@@ -138,25 +138,24 @@ div.dashboard_container .custom-tooltip {
   padding: 4px;
 }
 
-.nav-icon {
-  filter: var(--fill-accordion-button);
-  width: 24px;
-  height: 24px;
-}
-
-.nav-icon-text {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  gap: 6px;
-  justify-content: center;
-}
-
-.icon-button.btn.btn-primary.active {
-  background-color: var(--state-overlays-selected);
+.icon-button.btn.btn-primary.active .material-symbols-outlined {
+  color: var(--text-active);
 }
 
 .loading-container {
   height: 100%;
   width: 100%;
 }
+
+.mantine-15jpsi0 {
```

Comment:
Why do we have this weird class name here?

Reply:
Weird class name corrected.